README.md

```diff
@@ -133,10 +133,33 @@ spec:
 
 5. Deploy Che operator:
 
+
 ```bash
 $ chectl server:deploy --installer=olm --platform=<CHECTL_SUPPORTED_PLATFORM> --catalog-source-yaml <PATH_TO_CUSTOM_CATALOG_SOURCE_YAML> --olm-channel=nightly --package-manifest-name=eclipse-che-preview-<openshift|kubernetes>
 ```
 
+### Deploy stable Che operator in Cluster Wide Availability
+
+Eclipse Che introduce a new channel which install Eclipse Che in AllNamespace mode with Devworkspace Operator like a OLM dependency.
+More info about DevWorkspace Operator can be found [here](https://github.com/operator-framework/operator-sdk).
+
+Before installing Eclipse Che using channel `stable-all-namespaces`  we need to consider the following:
+
+* It is not possible to have Eclipse Che installed in single Namespace (currently the default one) and then try to install Che in All
+* Namespace mode using the new channel stable-all-namespaces.
+* To update to stable-all-namespaces channel you need first to remove all subscriptions created for che installed from nightly or stable
+channels. IMPORTANT: Removing subscriptions doesn’t mean Eclipse Che operands(che-server, keycloak or roles) will be removed from the cluster.
+* DevWorkspace engine will be by default enabled in the new channel.
+* In case if you have already installed Che with devWorkspace enabled from channels nightly or stable you need to uninstall DevWorkspace completely from the cluster and then install the Che Operator using stable-all-namespaces channel.
+* Channel under development mode
```

Comment:
I have this for that case:
```
* In case if you have already installed Che with devWorkspace enabled from channels nightly or stable you need to uninstall DevWorkspace completely from the cluster and then install the Che Operator using stable-all-namespaces channel.
```

By default DevWorkspace engine will be by default enabled in the new channel.